### PR TITLE
feat: support global keyframes

### PR DESCRIPTION
**Pull request by tzachbon**
How to create global keyframes.

### CODE REVIEW

File: docs/references/keyframes.md [hunk 1]
```diff
@@ -117,3 +117,22 @@ import { classes, keyframes } from './entry.st.css';
 </div>
 ```
 
+## Global keyframes
+
+To create global (unscoped) keyframes, **Stylable** provides a function that takes the global keyframes name as a parameter.
```
Comment (tomrav) [inline docs/references/keyframes.md:122]:
```suggestion
To create global (unscoped) keyframes, **Stylable** provides the `stGlobal` function which accepts a global keyframe name as its parameter.
```

**Review by tomrav — APPROVED** [review]
Left a small comment, otherwise looks good.

File: docs/references/keyframes.md (hunk 1 above)
Comment (barak007) [reply docs/references/keyframes.md:122]: not sure about the `*the* stGlobal`

Comment (tzachbon) [reply docs/references/keyframes.md]: I can agree with @barak007 on this one.

**Pull request merged**